### DISCUSSION TITLE
Changes for 1.8" 128x160 display

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_midi_sampler#44] [Q&A]
@marcel-licence  I am using a 1.8" 128x160 display with this project, and is working. But the texts and images are like a 80x160 display because that is how you programmed it. Can you please help me? Where should I edit the code to make it compatible for 128x160 display and get bigger texts and images?

**Comment by marcel-licence** [comment]
Display_module.ino and screens.ino are doing the graphical stuff in this project

**Reply by souravlal** [reply]
ok
